packages/database/src/sync/buildEncounterLinkedLookupFilter.ts

```diff
@@ -5,32 +5,25 @@ import {
 import { buildSyncLookupSelect } from './buildSyncLookupSelect';
 import type { Model } from '../models/Model';
 
-export type Options = {
-  extraSelects?: Record<string, string>; // extra selects to attach to the lookup select
-  extraJoins?: JoinConfig[]; // extra joins needed to traverse between this model and the encounters table
-};
-
 /**
- * Helper function to determine if a facility_id should be populated in sync lookup
+ * Helper to determine if a facility_id should be populated in sync lookup
  * Only populates facility_id when the encounter is from a sensitive facility
  * This ensures sensitive encounters are only synced to their originating facility
  */
-export function addSensitiveFacilityIdIfApplicable() {
-  return `
+const addSensitiveFacilityIdIfApplicable = `
```

Comment:
Caps snake case is our usual global const convention. That's why I was meaning it would be less readable, it's definitely not as pleasant to look at, but we'd better stick with the pattern!